lib/matplotlib/testing/conftest.py

```diff
@@ -125,3 +125,46 @@ def test_imshow_xarray(xr):
 
     xr = pytest.importorskip('xarray')
     return xr
+
+
+@pytest.fixture
+def _text_placeholders(monkeypatch):
```

Comment:
```suggestion
def _text_placeholders(monkeypatch):
    """
    Replace texts with a placeholder rectangle.
    
    The rectangle size only depends on the font size and the number of
    characters. It is thus insensitive to font properties and rendering
    details. This should be used for tests that depend on text geometries
    but not the actual text rendering, e.g. layout tests.
```

Reply:
Also, do we need to make this private? Little of the other stuff in `matplotlib.testing` is private. I could imagine that this is useful for downstream libraries as well. I don't think we would massively limit us if this is public. Worst case, we could create a new fixture under a different name and deprecate the old one, if we need to make a massively breaking change.